metricbeat/docs/fields.asciidoc

```diff
@@ -1192,6 +1192,83 @@ type: long
 Last updated
 
 
+[float]
+== osd_df fields
+
+ceph osd disk usage information
+
+
+
+[float]
+=== `ceph.osd_df.id`
+
+type: long
+
+osd node id
+
+
+[float]
+=== `ceph.osd_df.name`
+
+type: text
+
+osd node name
+
+
+[float]
+=== `ceph.osd_df.device_class`
+
+type: keyword
+
+osd node type, illegal type include hdd, ssd etc.
+
+
+[float]
+=== `ceph.osd_df.total_byte`
```

Comment:
This should probably be `ceph.osd_df.total.bytes`. have a look at the naming conventions: https://www.elastic.co/guide/en/beats/devguide/current/event-conventions.html  Same applies to the fields below.

Reply:
ok, I will fix it.